CONTRIBUTING.md

```diff
@@ -1,17 +1,17 @@
-# Contributing to OpenEBS Litmus
+# Contributing to OpenEBS e2e-tests
 
-OpenEBS Litmus is an Apache 2.0 Licensed project and uses the standard GitHub pull requests process to review and accept contributions.
+OpenEBS e2e-tests is an Apache 2.0 Licensed project and uses the standard GitHub pull requests process to review and accept contributions.
 
-There are several areas of Litmus that could use your help. For starters, you could help in improving the sections in this document by either creating a new issue describing the improvement or submitting a pull request to this repository. 
+There are several areas of e2e that could use your help. For starters, you could help in improving the sections in this document by either creating a new issue describing the improvement or submitting a pull request to this repository. 
 
 * If you are a first-time contributor, please see [Steps to Contribute](#steps-to-contribute).
-* If you would like to suggest new tests to be added to litmus, please go ahead and [create a new issue](https://github.com/openebs/litmus/issues/new) describing your test. All you need to do is specify the workload type and the operations that you would like to perform on the workload.
+* If you would like to suggest new tests to be added to this repo, please go ahead and [create a new issue](https://github.com/openebs/litmus/issues/new) describing your test. All you need to do is specify the workload type and the operations that you would like to perform on the workload.
 * If you would like to work on something more involved, please connect with the OpenEBS Contributors. See [OpenEBS Community](https://github.com/openebs/openebs/tree/master/community)
 * If you would like to make code contributions, all your commits should be signed with Developer Certificate of Origin. See [Sign your work](#sign-your-work). 
 
 ## Steps to Contribute
 
-* Find an issue to work on or create a new issue. The issues are maintained at [openebs/litmus](https://github.com/openebs/litmus/issues). You can pick up from a list of [good-first-issues](https://github.com/openebs/litmus/labels/good%20first%20issue).
+* Find an issue to work on or create a new issue. The issues are maintained at [openebs/e2e-tests](https://github.com/openebs/e2e-tests/issues). You can pick up from a list of [good-first-issues](https://github.com/openebs/litmus/labels/good%20first%20issue).
```

Comment:
Here also need to change the repo URL:- `https://github.com/openebs/e2e-tests/labels/good%20first%20issue`